### ISSUE TITLE
Cannot expand ZIP build/outputs/aar/api--release.aar

### ISSUE THREAD

**Issue by george-vlahakis** [kezong/fat-aar-android#242]
Anyone else getting this? 
In essence I have an embed project(...) in my library project which consumes another library project.

I thought it had something to do with my library but cannot fing anything wrong.

Obviously it builds fine.

PS: Normall we use 

```
    libraryVariants.all { variant ->
        variant.outputs.all {
            outputFileName
```

to set the library binary out filename.

**Comment by kezong** [comment]
Do you add flavor config in library module's `build.gradle`?

**Comment by george-vlahakis** [comment]
No flavors in project "api"

**Comment by kezong** [comment]
Try upgrade plugin to 1.3.1.

**Comment by george-vlahakis** [comment]
Tried that. Doesnt work at all. Error when I upgrade to 1.3.1:

```

Cannot access first() element from an empty List

Caused by: java.util.NoSuchElementException: Cannot access first() element from an empty List
	at com.kezong.fataar.FlavorArtifact$_getBundleTask_closure1.doCall(FlavorArtifact.groovy:117)
	at com.kezong.fataar.FlavorArtifact.getBundleTask(FlavorArtifact.groovy:105)
	at com.kezong.fataar.FlavorArtifact.createFlavorArtifact(FlavorArtifact.groovy:29)
	at com.kezong.fataar.FlavorArtifact$createFlavorArtifact.call(Unknown Source)
	at com.kezong.fataar.FatAarPlugin$_dealUnResolveArtifacts_closure7.doCall(FatAarPlugin.groovy:143)
	at com.kezong.fataar.FatAarPlugin.dealUnResolveArtifacts(FatAarPlugin.groovy:135)
	at com.kezong.fataar.FatAarPlugin$_doAfterEvaluate_closure3$_closure8.doCall(FatAarPlugin.groovy:65)
	at com.kezong.fataar.FatAarPlugin$_doAfterEvaluate_closure3.doCall(FatAarPlugin.groovy:58)
	at org.gradle.util.ClosureBackedAction.execute(ClosureBackedAction.java:70)
	at org.gradle.util.ConfigureUtil.configureTarget(ConfigureUtil.java:154)
	at org.gradle.util.ConfigureUtil.configure(ConfigureUtil.java:105)
	at org.gradle.util.ConfigureUtil$WrappedConfigureAction.execute(ConfigureUtil.java:166)
	at org.gradle.api.internal.DefaultDomainObjectCollection.all(DefaultDomainObjectCollection.java:158)
	at org.gradle.api.internal.DefaultDomainObjectCollection.all(DefaultDomainObjectCollection.java:192)
	at org.gradle.api.DomainObjectCollection$all.call(Unknown Source)
	at com.kezong.fataar.FatAarPlugin.doAfterEvaluate(FatAarPlugin.groovy:55)
	at com.kezong.fataar.FatAarPlugin$_apply_closure1.doCall(FatAarPlugin.groovy:40)
	at org.gradle.configuration.internal.DefaultListenerBuildOperationDecorator$BuildOperationEmittingClosure$1$1.run(DefaultListenerBuildOperationDecorator.java:181)
	at org.gradle.configuration.internal.DefaultUserCodeApplicationContext.reapply(DefaultUserCodeApplicationContext.java:58)
	at org.gradle.configuration.internal.DefaultListenerBuildOperationDecorator$BuildOperationEmittingClosure$1.run(DefaultListenerBuildOperationDecorator.java:176)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:402)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:394)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor$1.execute(DefaultBuildOperationExecutor.java:165)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:250)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:158)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:92)
	at org.gradle.configuration.internal.DefaultListenerBuildOperationDecorator$BuildOperationEmittingClosure.doCall(DefaultListenerBuildOperationDecorator.java:173)
	at org.gradle.listener.ClosureBackedMethodInvocationDispatch.dispatch(ClosureBackedMethodInvocationDispatch.java:40)
	at org.gradle.listener.ClosureBackedMethodInvocationDispatch.dispatch(ClosureBackedMethodInvocationDispatch.java:25)
	at org.gradle.internal.event.AbstractBroadcastDispatch.dispatch(AbstractBroadcastDispatch.java:42)
	at org.gradle.internal.event.BroadcastDispatch$SingletonDispatch.dispatch(BroadcastDispatch.java:230)
	at org.gradle.internal.event.BroadcastDispatch$SingletonDispatch.dispatch(BroadcastDispatch.java:149)
	at org.gradle.internal.event.AbstractBroadcastDispatch.dispatch(AbstractBroadcastDispatch.java:58)
	at org.gradle.internal.event.BroadcastDispatch$CompositeDispatch.dispatch(BroadcastDispatch.java:324)
	at org.gradle.internal.event.BroadcastDispatch$CompositeDispatch.dispatch(BroadcastDispatch.java:234)
	at org.gradle.internal.event.ListenerBroadcast.dispatch(ListenerBroadcast.java:140)
	at org.gradle.internal.event.ListenerBroadcast.dispatch(ListenerBroadcast.java:37)
	at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:93)
	at com.sun.proxy.$Proxy37.afterEvaluate(Unknown Source)
	at org.gradle.configuration.project.LifecycleProjectEvaluator$NotifyAfterEvaluate$1.execute(LifecycleProjectEvaluator.java:190)
	at org.gradle.configuration.project.LifecycleProjectEvaluator$NotifyAfterEvaluate$1.execute(LifecycleProjectEvaluator.java:187)
	at org.gradle.api.internal.project.DefaultProject.stepEvaluationListener(DefaultProject.java:1424)
	at org.gradle.configuration.project.LifecycleProjectEvaluator$NotifyAfterEvaluate.run(LifecycleProjectEvaluator.java:196)```

**Comment by kezong** [comment]
I tried to fix it in 1.3.3, you can try it.

**Comment by george-vlahakis** [comment]
Tried 1.3.3 and the release build generates a fat aar as expected. Have not tested it per say yet, but at least the build logic works.

Thank you @kezong !
